tool-testsuite/test/org/antlr/v4/test/tool/TestToolSyntaxErrors.java

```diff
@@ -60,6 +61,16 @@ public void testSetUp() throws Exception {
 		super.testSetUp();
 	}
 
+	@Test
+	public void AllErrorCodesDistinct() {
+		ErrorType[] errorTypes = ErrorType.class.getEnumConstants();
+		for (int i = 0; i < errorTypes.length; i++) {
```

Comment:
I think my solution is better because of `assertNotEquals` fails on the certain error code. It's more clear that length comparison.